frontendFile/src/main/java/com/virtuslab/gitmachete/frontend/file/highlighting/MacheteSyntaxHighlighterFactory.java

```diff
@@ -0,0 +1,16 @@
+package com.virtuslab.gitmachete.frontend.file.highlighting;
+
+import com.intellij.openapi.fileTypes.SyntaxHighlighter;
+import com.intellij.openapi.fileTypes.SyntaxHighlighterFactory;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.vfs.VirtualFile;
+import org.checkerframework.checker.guieffect.qual.UI;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+@UI
```

Comment:
Hmm similar to the comment on a Checker stub, do we want that factory to run on UI thread? Although I think that's less brittle than running the highlighter itself on UI thread